tests/lib/assertions/classic-model-rev1-22.assert

```diff
@@ -38,13 +38,13 @@ snaps:
 timestamp: 2023-11-09T07:50:10+00:00
 sign-key-sha3-384: 7qWG-Uwck6Dji43a3Z8ZZrm7rAziZAch3xf76iFvqe4GaD0LI7U9lYPWMSJAsEgu
 
-AcLBcwQAAQoAHRYhBGESvKlz1RXG1IBOC0MdJKf2hr9ABQJlnX8XAAoJEEMdJKf2hr9AZ9wP/3gt
-bYcC60r4r4Vtnsy2K3L851vtyI+Lm+S/7X4dXtQhHHXTEkIbYUBnTyEtI2BDTQ8LF0LRuqK7Mb9n
-E4h+rJzxlp3X3c97FzRCw1ctaWcCQi97+VY0jmGW5Ytp0hSY0+f5rcQYXMtEGzpNGNOnwb6hS5MA
-pp5LmipRNHuC6Ss5assT2F6iCd+5aUnvfMkVlv+cM8EZ7Yd+SGGjtBTUryk98JqP29bdYIH4QvpQ
-17Y8DUKkCBzqyco3VT2Wk51Fbl+BKrL/vIqEWmSREj6MmhpJ9DSAptU9HmVj5HAY6T362zOwz7Py
-6OMy/Dxyf1W/eUm7CuQZ+uybnqZD5pCulbI5WX72kVNqjCJUGJWA61ApKVjMbWuUtIS9HuP1kDhy
-a7E6tB7C3NP7yr4qfG78euPW4bH3kzObvCPY0vf3/v0xXM7jho0fFlX+lg60Wq46haXHU/Jn7MTy
-D8HACuT9kY5YTQ9yiUQsMvon3CdA9Z7ViF0ff6+xrwnjAlqCIBop5AMLe4/gGgSa+njA4esHO4Ne
-j9ErTHIA+79CQjKuEgHCZ6xvvoNQuUdhonSf7xNvY+gCDfFotw53yGmJQy6klIAg1ufVjCNbnfqd
-7dGhTM0h3j7GG6sH3rcmdUyxQIgaU8E8fCQbL0jcLGIKwhPv7f9taxQAxlyZKDqsFX22eFU5
+AcLBcwQAAQoAHRYhBGESvKlz1RXG1IBOC0MdJKf2hr9ABQJm2xVIAAoJEEMdJKf2hr9AFZcP/iNS
+P0VYXtrYRP4SXJnSws9MSvosF1Q6Zj4vKAddKAlswnu87/e6KaKw/xlFg5MfcgoWAjruU662sxPJ
+1X1f579LnPFm7OwnG5IiGKeK5LOtn1T9JADNwLYKQyXlwfiLaBaqPQ0PsXSYxx/v3AW7421sHK2V
+fHtUWSag/olAW+LqerRJsrgSg4UjxpItboej9wsOWzU6rCW9nDlSP4d3FsaYTlasurXUVegtW+XE
+UDdcQ07KLkpJk5PNIa9Ch55gwnMRRhI9eXhibh1L9xDQLtRigzxWQh7W8F08/WoFsywmLdihfUUM
+JUgT7z7t6MmahYVUy3vz7Fhtj5UZCET4UL89/NQ+XUVvjYcqBz0GRFXJ02yqVQNwBe8BvC8I1GQ9
+eQao+hEuJ2Zz51GV1IVkV0VITgo0APWqnF8DHAvTNlJ3rlViJHrbvW7qLxaIvv061Rj9IGE/2RNk
+nW/WZQw5XGPYLz5q00dKiW5/5yTcHkFFiCZBcJws2vKVtZdgr7ggkbApl3UIbWNuTc9Fz6UniVj5
+0MFFGS38bvfngTrHw9fn00zzgtaNGpsN9JmK3baC1qOU9c159vP3nlNkaDe21mpRHU6BX2tacCM9
+nQJetYmBebuUHXq/wl/T6fJNvQk42+u6b+68kx/KAl3udwfsiUF48WiRsHcChqNZQhpRsoCm
```

Comment:
I believe assertions are supposed to have this newline here, not sure if that will prevent things from working or not.